sdk/ai/ai-inference/review/ai-inference-rest.api.md

```diff
@@ -0,0 +1,449 @@
+    };
+}
+
+// @public (undocumented)
+export interface GetChatCompletionsDefaultHeaders {
+    "x-ms-error-code"?: string;
+}
+
+// @public (undocumented)
+export interface GetChatCompletionsDefaultResponse extends HttpResponse {
+    // (undocumented)
+    body: ErrorResponse;
+    // (undocumented)
+    headers: RawHttpHeaders & GetChatCompletionsDefaultHeaders;
+    // (undocumented)
+    status: string;
+}
+
+// @public (undocumented)
+export interface GetChatCompletionsHeaderParam {
+    // (undocumented)
+    headers?: RawHttpHeadersInput & GetChatCompletionsHeaders;
+}
+
+// @public (undocumented)
+export interface GetChatCompletionsHeaders {
+    "unknown-parameters"?: UnknownParams;
+}
+
+// @public (undocumented)
+export type GetChatCompletionsParameters = GetChatCompletionsHeaderParam & GetChatCompletionsBodyParam & RequestParameters;
+
+// @public (undocumented)
+export interface GetEmbeddings {
+    post(options?: GetEmbeddingsParameters): StreamableMethod<GetEmbeddings200Response | GetEmbeddingsDefaultResponse>;
+}
+
+// @public
+export interface GetEmbeddings200Response extends HttpResponse {
+    // (undocumented)
+    body: EmbeddingsResultOutput;
+    // (undocumented)
+    status: "200";
+}
+
+// @public (undocumented)
+export interface GetEmbeddingsBodyParam {
+    // (undocumented)
+    body?: {
+        input: string[];
+        dimensions?: number;
+        encoding_format?: EmbeddingEncodingFormat;
+        input_type?: EmbeddingInputType;
+    };
+}
+
+// @public (undocumented)
+export interface GetEmbeddingsDefaultHeaders {
+    "x-ms-error-code"?: string;
+}
+
+// @public (undocumented)
+export interface GetEmbeddingsDefaultResponse extends HttpResponse {
+    // (undocumented)
+    body: ErrorResponse;
+    // (undocumented)
+    headers: RawHttpHeaders & GetEmbeddingsDefaultHeaders;
+    // (undocumented)
+    status: string;
+}
+
+// @public (undocumented)
+export interface GetEmbeddingsHeaderParam {
+    // (undocumented)
+    headers?: RawHttpHeadersInput & GetEmbeddingsHeaders;
+}
+
+// @public (undocumented)
+export interface GetEmbeddingsHeaders {
+    "unknown-parameters"?: UnknownParams;
+}
+
+// @public (undocumented)
+export type GetEmbeddingsParameters = GetEmbeddingsHeaderParam & GetEmbeddingsBodyParam & RequestParameters;
+
+// @public (undocumented)
+export interface GetImageEmbeddings {
+    post(options?: GetImageEmbeddingsParameters): StreamableMethod<GetImageEmbeddings200Response | GetImageEmbeddingsDefaultResponse>;
+}
+
+// @public
+export interface GetImageEmbeddings200Response extends HttpResponse {
+    // (undocumented)
+    body: EmbeddingsResultOutput;
+    // (undocumented)
+    status: "200";
+}
+
+// @public (undocumented)
+export interface GetImageEmbeddingsBodyParam {
+    // (undocumented)
+    body?: {
+        input: Array<EmbeddingInput>;
+        dimensions?: number;
+        encoding_format?: EmbeddingEncodingFormat;
+        input_type?: EmbeddingInputType;
```

Comment:
`EmbeddingInputType` is defined as follows: `type EmbeddingInputType = string | "text" | "query" | "document";`. I just wanted to double check, are these types applicable to the /images/embeddings API as well?

Reply:
removing embeddings apis and types from v1